src/main/java/androidx/view/ViewGroup.kt

```diff
@@ -114,3 +116,12 @@ fun ViewGroup.MarginLayoutParams.updateMarginsRelative(
     marginEnd = end
     bottomMargin = bottom
 }
+
+/**
+ * Inflates the given layoutId into the ViewGroup.
```

Comment:
Can you be more specific and explain that the inflated view is added as a child to this ViewGroup?